FurkannOzbek/personal-site-react
# [FEATURE] Initial settings & Landing Page

## Description

This pull request includes the following changes:

- Modified the default settings of the project.
- Created a landing page with new text and updated styles.


## Type of Change


## How to Test

To test the changes, follow these steps:

1. Ensure the project is running locally by starting the development server.
2. Navigate to the landing page.
3. Verify that the new text appears correctly on the page.
4. Check the updated styles and ensure they render correctly on different screen sizes.


## Checklist

- [x] I have reviewed my code and ensured no sensitive data is exposed

## Screenshots

**Desktop View:**

![LandingPageText](https://github.com/user-attachments/assets/899aac05-32f7-4197-b460-570338ae727c)
